product-catalog/app/productcatalog/models/RichShippingRate.java

```diff
@@ -0,0 +1,15 @@
+package productcatalog.models;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.shippingmethods.ShippingRate;
+
+public class RichShippingRate extends Base {
```

Comment:
:+1: for consitent naming convention. If we wrap jvm's ```ShippingRate``` then i would vote for ```CustomShippingRate ``` or maybe ```ShippingRateData```